sdk/python/feast/field.py

```diff
@@ -14,21 +14,25 @@
 
 from feast.value_type import ValueType
 
-
 class Field:
     """
     High level field type. This is the parent type to both entities and
     features.
     """
 
-    def __init__(self, name: str, dtype: ValueType):
+    def __init__(self, name: str, dtype: ValueType, labels=None):
```

Comment:
Shouldnt `labels` have a type?

Reply:
It is of type dictionary it is handled on the initiation below, this is to make the `labels` Optional so the function signature doesn't change

Reply:
You can use Optional[Dict] there I think

Reply:
If what you mean is [typing.Optional](https://docs.python.org/3/library/typing.html)

I have tried to use it before, it is not behaving as `optional parameter` instead according to the documentation it is behaving as `optional value`

Reply:
It's not entirely clear what you mean. Can you please refer to this explanation of Optional type hints: https://stackoverflow.com/questions/51710037/how-should-i-use-the-optional-type-hint

Reply:
Okay, I guess after python 3.5 they are using the Optional aforementioned for type hinting. Have added it